extensions/stackdriver/stackdriver.cc

```diff
@@ -265,6 +268,19 @@ bool StackdriverRootContext::configure(size_t configuration_size) {
   }
 
   if (!logger_ && enableAccessLog()) {
+    std::unordered_map<std::string, std::string> extra_labels;
+    if (config_.has_custom_log_config()) {
+      cleanupExpressions();
```

Comment:
Shouldn't we clean up expressions if access log disabled? What happens if the config is updated to remove access logging? onConfigure would be called twice.

Reply:
Also, it seems that logger_ would be set. Is it possible to re-configure SD?

Reply:
Thanks for catching..

I think reconfiguring logger would mean loosing the log entries that were in queue.. so either flush those entries and then reconfigure. There is a need now to do this as batch size can be changed in onConfigure...

Reply:
This issue doesn't seem to be specific to SD. Let's address this here and in stats separately.